app/gui/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -1,4 +1,12 @@
-//! This module defines the `Container` struct and related functionality.
+//! This module defines the `Container` struct and related functionality. This represent the view
+//! a visualisation int he graph editor and includes a visual box that contains the visualisaiton,
+//! a action bar that allows setting the visualisation type.
+//!
+//! The `[Container]` struct is responsible for managing the visualisation and action bar and
+//! providing a unified interface to the graph editor. This includes ensuring that the visualisation
+//! is correctly positioned, sized and layouted in its different `ViewState`s (which include the
+//! `Enabled`, `Fullscreen` and `Preview` states). Importantly, this also includes layer management
```

Comment:
You can put `ViewState` in [] (`[ViewState]`) making them link to the structure; then I think it's not necessary to list enum variants.